sdk/spring/spring-cloud-azure-autoconfigure/src/main/java/com/azure/spring/cloud/autoconfigure/implementation/compatibility/AzureSpringBootVersionVerifier.java

```diff
@@ -99,7 +99,10 @@ private boolean springBootVersionMatches() {
                 String fullyQualifiedClassName = this.supportedVersions.get(versionString);
 
                 if (classNameResolver.resolve(fullyQualifiedClassName)) {
-                    LOGGER.debug("Predicate for Spring Boot Version of [" + versionString + "] was matched");
+                    if (LOGGER.isDebugEnabled()) {
```

Comment:
I don't think this check is required, in this context, it does nothing

Reply:
Yes, we may not need this check in this context.